tests/bfd/test_bfd.py

```diff
@@ -7,6 +7,13 @@
     pytest.mark.topology('t1', 't1-lag', 't1-64-lag')
```

Comment:
Just mark topology "t1" is enough. Here the mark is about topology type, not topology name.
Both "t1", "t1-lag" and "t1-64-lag" belong to topology type "t1".

Reply:
fixed